source/docs/software/kinematics-and-odometry/intro-and-chassis-speeds.rst

```diff
@@ -9,6 +9,115 @@ The kinematics suite contains classes for differential drive, swerve drive, and
 ## What is odometry?
 Odometry involves using sensors on the robot to create an estimate of the position of the robot on the field. In FRC, these sensors are typically several encoders (the exact number depends on the drive type) and a gyroscope to measure robot angle. The odometry classes utilize the kinematics classes along with periodic user inputs about speeds (and angles in the case of swerve) to create an estimate of the robot's location on the field.
 
+## Why use kinematics and odometry?
+
+Teams should use WPILib's kinematics and odometry classes for several important reasons:
+
+### Enhanced Autonomous Performance
+
+**Accurate path following**: Kinematics and odometry enable your robot to precisely follow complex paths during autonomous. Instead of time-based driving ("go forward for 2 seconds"), your robot can navigate to specific field coordinates and execute sophisticated multi-step autonomous routines.
+
+**Position tracking**: Knowing where your robot is on the field allows you to:
+
+- Drive to specific game pieces or scoring locations
+- Dynamically adjust paths based on current position
+- Create reliable autonomous routines that work consistently
+
+**Self-correction**: Combined with vision systems (like AprilTag detection), odometry can be continuously updated to correct for drift and maintain accuracy throughout a match.
+
+### Field-Oriented Control
+
+For holonomic drivetrains (swerve and mecanum), field-oriented control dramatically improves driver experience:
+
+- Driver joystick inputs remain consistent regardless of robot orientation
+- Pushing forward on the joystick always moves toward the opposite alliance station
+- Robot can face any direction while moving in any direction
+- Significantly reduces driver cognitive load during teleoperation
+
+### Trajectory Generation and Following
+
+Kinematics integrates with WPILib's trajectory generation to create smooth, dynamically-constrained paths:
+
+- Generate realistic trajectories that respect your drivetrain's physical limits
+- Follow paths with feedback control for disturbance rejection
+- Combine with path planning tools like PathPlanner or Choreo
+
+### Common Use Cases
+
+**Autonomous routines**: Navigate to game pieces, score at specific locations, execute multi-step sequences
+
+**Vision-assisted driving**: Use AprilTags or other vision targets to update odometry and drive to precise field locations
+
+**Teleoperation enhancement**: Field-oriented control for intuitive driving of holonomic robots
+
+**Match strategy**: Plan routes that avoid defense, optimize cycle times, coordinate with alliance partners
+
+## When to use each drivetrain type
```

Comment:
It feels really out of place to talk about drivetrain selection in kinematics/odometry. The drivetrain selection should drive the code, and these characteristics have nothing to do with kinematics or odometry.